### PR TITLE
Testing infrastructure on main

### PR DESCRIPTION
**Pull request by beatthat**

### CODE REVIEW

File: mentor_classifier/tests/helpers.py [hunk 1]
```diff
@@ -5,6 +5,110 @@
...
+@dataclass
+class Topic:
+    name: str
+
+
+@dataclass
+class Question:
+    _id: str
+    question: Optional[str] = None
+    type: Optional[str] = None
```
Comment (beatthat) [inline mentor_classifier/tests/helpers.py:35]: in python, you almost never want to use `Optional[str]`, instead use empty string, e.g.

```python
@dataclass
class Question:
    _id: str
    question = ""
```

...remember that python has an expansive view of `falsy`ness...so

```python
bool("")  # False
bool([])  # False
bool({})  # False
```


question

File: mentor_classifier/tests/helpers.py [hunk 2]
```diff
@@ -5,6 +5,110 @@
...
+        question_ref = Question(_id=mentor_question.question_id)
+        topic = Topic(mentor_question.topic)
+        question_and_topics = QuestionAndTopics(question=question_ref, topics=[topic])
+        if topic not in self.topics:
+            self.topics.append(topic)
+        self.questions.append(question_and_topics)
+        question = Question(
+            _id=mentor_question.question_id,
+            question=mentor_question.question,
+            type=None,
```
Comment (beatthat) [inline mentor_classifier/tests/helpers.py:74]:
```python
type="", 
name=""
```

File: mentor_classifier/tests/helpers.py [hunk 3]
```diff
@@ -5,6 +5,110 @@
...
+            name=None,
+            paraphrases=mentor_question.paraphrases,
+        )
+        answer = Answer(
+            _id="A" + str(len(self.answers) + 1),
```
Comment (beatthat) [inline mentor_classifier/tests/helpers.py:79]: always prefer f strings over + concat

```python
_id = f"A{str(len(self.answers) + 1)}"
```

File: mentor_classifier/tests/helpers.py [hunk 4]
```diff
@@ -5,6 +5,110 @@
...
+            mentor_question = parse_mentor_question(line)
+            result.add_mentor_question(mentor_question)
+
+    return result
+
+
+def parse_mentor_question(csv_line: str) -> MentorQuestion:
+    columns = csv_line.split(",")
+    paraphrases = []
+    if columns[2] != "":
```
Comment (beatthat) [inline mentor_classifier/tests/helpers.py:103]: always prefer falsy check

```python
if not columns[2]:
```

...but here you should not use this `if` stmt at all, just do

```python
paraphrases = columns[2].split("|") if columns[2] else []
```

**Review by beatthat — COMMENTED** [review]
I think looks good, just inline comments on python style

**Review by dan100110 — APPROVED** [review]

**Review by dan100110 — APPROVED** [review]

**Pull request merged**